### ISSUE TITLE
Add shortcuts to change popular preferences for things from the right-click context menu (for example, hide Open Editors)

### ISSUE THREAD

**Issue by Duckers** [Microsoft/vscode#51037]
This issue might be seen as a variant of https://github.com/Microsoft/vscode/issues/3355 but instead of introducing a new "mother of all settings view", I propose adding quick access to popular settings in the right-click context menu of things.

For example, when I was new to VS Code I was instantly annoyed by the Open Editors panel in the Explorer. For me personally this adds zero value (this info is already provided to me by the horizontal tabs), is visual noise and eats up precious screen real-estate that I like to be reserved for the file tree.

My personal taste aside, my gut-instinct when I was new to VS Code was to right click on the Open Editors header bar in the explorer, looking for a way to get rid of the thing. Not finding any clues there was frustrating, leading me to start thinking that this was a mandatory feature. I had to google around and it turns out the setting to control this thing is not very intuitively named. It also meant my first experience with VS Code involved having to pull out and get familiar with the user preferences just for this.

In short: Please add something to the right-click context menu that allows me to easily hide the things i click on, when that's an option. Perhaps give me an alert afterwards telling me how to turn it back on again afterwards (as I don't want any buttons around stealing screen estate "just in case" i want to toggle it back on).

The Open Editors panel is just one simple example of a principle that I think can be applied to many features within VSCode. For example, right-clicking anywhere in the explorer view could offer the options to toggle common features on/off in the context menu.

**Comment by Duckers** [comment]
Nevermind, in the latest version of Code, this seems to have been already implemented :)
